src/cmd-buildextend-aws

```diff
@@ -68,12 +69,22 @@ def run_ore():
         ore_args.extend(['--grant-user', user])
     print("+ {}".format(subprocess.list2cmdline(ore_args)))
     ore_data = json.loads(subprocess.check_output(ore_args))
+    checksum = sha256sum_file(tmp_img_aws_vmdk)
+    size = os.path.getsize(tmp_img_aws_vmdk)
+    os.rename(tmp_img_aws_vmdk, f"{builddir}/{aws_vmdk_name}")
     shutil.rmtree(tmpdir)
     # This matches the Container Linux schema:
     # https://stable.release.core-os.net/amd64-usr/current/coreos_production_ami_all.json
     ami_data = {'name': args.region,
                 'hvm': ore_data['HVM']}
     buildmeta['amis'] = [ami_data]
+    buildmeta['images'].update({
+        'aws': {
+            'path': aws_vmdk_name,
+            'sha256': checksum,
+            'size': str(size)
```

Comment:
I followed the example of the other cmd-buildextends here, though it seems odd to stringify an integer for JSON.

Reply:
That does seem like a mistake...we could probably fix it?  I doubt anything is parsing it.

On the other hand, we would have to use strings to represent disk images larger than [the maximum safe value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) ...you may think no one is going to be building custom FCOS-derived images larger than 9PB but...it's the internet, someone might think it's a challenge...

Reply:
I do think that.  :smile:  Updated, and I'll PR the others.